### DISCUSSION TITLE
Kindly add the Force Set Device for TextEncoder!

### DISCUSSION THREAD

**Discussion by Iory1998** [lllyasviel/stable-diffusion-webui-forge#1753] [Ideas]
Regarding the Flux models, in ComfyUI, there is the feature to set/force loading text-encoders to the CPU rather the GPU, which helps with Vram management a lot. It does impact speed a bit, but it's a about a few seconds more, but the benefit is that I can use higher precision models and save time by unloading the text-encoders from Vram each time there is a change in the model or LoRA.

![image](https://github.com/user-attachments/assets/ddee1018-691d-41f1-b429-0cc8ac849756)

**Comment by lllyasviel** [comment]
you should move this PR to https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/1591#discussioncomment-10493377 -> https://github.com/Juqowel/GPU_For_T5

**Reply by Iory1998** [reply]
Thank you. I did that.
